<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCodeWindow</class>
 <widget class="QDialog" name="GCodeWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>370</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="teGcode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveGcodeTe">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icons/disk.png</normaloff>:/icons/disk.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Add G-Code</string>
   </property>
   <widget class="QPushButton" name="btnAddGcode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Add Gcode</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbGcodeCmd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>G01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M204</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbX">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leXValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leYValue">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leZValue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbZ">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVValue">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbV">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbA">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <zorder>lbA</zorder>
   <zorder>lbV</zorder>
   <zorder>lbZ</zorder>
   <zorder>lbY</zorder>
   <zorder>btnAddGcode</zorder>
   <zorder>cmbGcodeCmd</zorder>
   <zorder>lbX</zorder>
   <zorder>leXValue</zorder>
   <zorder>leYValue</zorder>
   <zorder>leZValue</zorder>
   <zorder>leVValue</zorder>
   <zorder>leAValue</zorder>
  </widget>
  <widget class="QComboBox" name="cmbProgramList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Program 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Program 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Program 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Program 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Program 5</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnFormat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Format</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
